### ISSUE TITLE
Error in action testing IW via Acess database

### ISSUE THREAD

**Issue by bl-young** [USEPA/LCIAformatter#85]
Actions can not run due to the following [error](https://github.com/USEPA/LCIAformatter/actions/runs/3017720932):

> crazy-max/ghaction-chocolatey@v1 is not allowed to be used in USEPA/LCIAformatter. Actions in this workflow must be: within a repository that belongs to your Enterprise account, created by GitHub, or matching the following: cloud-gov/*, softprops/turnstyle@v1, zaproxy/*, r-lib/actions/*, docker/*.

Looks like the installation of ms access with the current setup is not allowed

**Comment by bl-young** [comment]
@WesIngwersen we can remove this dependency and not include any further testing with IW unless you have other ideas

**Comment by WesIngwersen** [comment]
Yes I agree we attempted numerous times to make this feasible. We can keep an archived version of the LCIAmethod and be in touch with the provider as its needed by users to communicate the issues with this data format and encourage another that's more friendly to automation

**Comment by bl-young** [comment]
@WesIngwersen this is actually an issue related to the package we use for the github action itself.

Github lists a procedure that you can do to add a package.
In our case we need to add crazy-max/ghaction-chocolatey
see the numbered steps [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#allowing-select-actions-and-reusable-workflows-to-run)

I do not have permissions for this.

![image](https://user-images.githubusercontent.com/44471635/200854043-0c2c2e60-7e26-496b-be10-f743ef2e41ae.png)
